app/actors/hyrax/actors/file_set_actor.rb

```diff
@@ -138,7 +137,8 @@ def ability
       end
 
       def build_file_actor(relation)
-        file_actor_class.new(file_set, relation, user, use_valkyrie: use_valkyrie)
+        fs = use_valkyrie ? file_set.valkyrie_resource : file_set
+        file_actor_class.new(fs, relation, user, use_valkyrie: use_valkyrie)
```

Comment:
Other than testing, do we ever go through the actor stack with valkyrie resources?  We did when this was written.  But now that transactions are called for updating when the controller sees a resource, I'm wondering about the code in the actor stack that checks use_valkyrie.  Should it be deprecated pointing to the transactions as the alternative?

Reply:
deprecating seems okay, but i'm not sure i see any need. it's getting reworked now, with the removal of this Wings special handling, but i think it's unlikely to need attention again until we're ready to yank Wings entirely. there's every reason to imagine we'd deprecate/remove the whole actor stack by then.